twitter_like_app/src/components/home_component/Home.jsx

```diff
@@ -0,0 +1,43 @@
+import React,{Component} from "react";
+import './Home.css'
+
+class Home extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            posts : []
+        }
+    }
+
+    componentDidMount(){
+        fetch('http://localhost:3000/posts')
+        .then(res => res.json())
+        .then(result => this.setState({posts : result}))
+    }
+
+    render() {
+
+        return (
+            <div className='main'>
+                <h1>Twitter Like app</h1>
+                {this.state.posts.map((post) => {
+                    return (
+                        
+                        <div className='posts' key={post['id']}>
+                            <div className="posts_title">{post['title']}</div>
+                            <div className='likes'>💚{post['likes']}</div>
+                            <div className='posts_body'>{post['body']}</div>
+                        </div>
```

Comment:
Make no sense since you receive single `post` from 23d line from map function. So why every post has this wrapper called `posts`?
Also consider separating these components. This is `Home` page, and this is singular post, that will be modified later. It should be the separate component.